backend/routes/group.py

```
@@ -0,0 +1,53 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+from collections import defaultdict
+from models.user import User
+from models.post import Post
+from schemas.university import UniversityPage, Member, UniversityPost
```

Ruff (0.8.2)

`schemas.university.UniversityPost` imported but unused

Remove unused import: `schemas.university.UniversityPost`

(F401)